maintainers/maintainer-list.nix

```diff
@@ -22384,6 +22384,11 @@
     github = "thilobillerbeck";
     githubId = 7442383;
   };
+  thinnerthinker = {
```

Comment:
changes to the maintainer file should be in a separate commit titled `thinnerthinker`. That commit should preceed the commit adding the package

Reply:
thanks!